packaging/hudi-flink-bundle/pom.xml

```diff
@@ -149,6 +150,10 @@
                   <include>org.apache.hbase:hbase-protocol</include>
                   <include>org.apache.htrace:htrace-core</include>
                   <include>commons-codec:commons-codec</include>
+
```

Comment:
> @umehrot2 just one comment on avoiding the `hoodie-aws` dependency in `client-common`. otherwise LGTM given you have already tested this out e2e. let me know what you and @zhedoubushishi think

```HoodieWriteConfig``` is located in ```client-common```, I added ```HoodeAWSConfig``` there:
```
writeConfig.setDefaultOnCondition(!isAWSConfigSet,
          HoodieAWSConfig.newBuilder().fromProperties(writeConfig.getProps()).build());
```
That's the reason why ```client-common``` is depend on ```hudi-aws```.